### ISSUE TITLE
"_curses.error: curs_set() returned ERR" when trying to open (most/any) CSV

### ISSUE THREAD

**Issue by jsvine** [saulpw/visidata#80]
After upgrading from `0.92` to `0.94`, I can't seem to open files with `vd`. I run into the same error on `0.93`.

## Setup

- OSX
- Python 3.6
- VisiData 0.94

## Command

```sh
vd path/to/my.csv
```

## Traceback

```
Traceback (most recent call last):
  File "[...]/bin/vd", line 39, in <module>
    main()
  File "[...]/bin/vd", line 36, in main
    visidata.run(sources)
  File "[...]/lib/python3.6/site-packages/visidata/vd.py", line 2079, in run
    ret = wrapper(cursesMain, sheetlist)
  File "[...]/lib/python3.6/site-packages/visidata/vd.py", line 1962, in wrapper
    return curses.wrapper(setupcolors, f, *args)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/curses/__init__.py", line 94, in wrapper
    return func(stdscr, *args, **kwds)
  File "[...]/lib/python3.6/site-packages/visidata/vd.py", line 1958, in setupcolors
    return f(stdscr, *args)
  File "[...]/lib/python3.6/site-packages/visidata/vd.py", line 2097, in cursesMain
    return vd().run(_scr)
  File "[...]/lib/python3.6/site-packages/visidata/vd.py", line 552, in run
    curses.curs_set(0)
_curses.error: curs_set() returned ERR
```

**Comment by saulpw** [comment]
@jsvine, which terminal program are you using?

**Comment by jsvine** [comment]
I get the same error in each of the three terminal environments I've tried:

- OSX Terminal (Native)
- iTerm2
- tmux within iTerm2

Also, if it helps, I'm running `vd` from within a `virtualenv` virtual environment, with only `visidata` installed.

**Comment by saulpw** [comment]
Okay, I don't know why we're not able to reproduce this, but this was just a cosmetic change, disabling the cursor from displaying in the main mode.  It'll be removed or wrapped in v0.95 (to be released this weekend).

**Comment by saulpw** [comment]
Should be fixed in v0.96
